doc/hands_on/hands_on_parquet.md

```diff
@@ -0,0 +1,240 @@
+
+So download this tool from the [GitHub releases](https://github.com/exasol/parquet-edml-generator/releases).
+
+Now you can run it locally on the command line:
+
+```shell
+java -jar parquet-edml-generator.jar books.parquet > mapping.json
+```
+
+Please note that you need to have our example file (`books.parquet`) in the same directory.
+
+The generator creates the following mapping:
+
+```json
+{
+  "source": "books.parquet",
+  "destinationTable": "BOOKS",
+  "description": "This mapping definition was auto-generated by the parquet-edml-generator.",
+  "addSourceReferenceColumn": false,
+  "mapping": {
+    "fields": {
+      "Authors": {
+        "toJsonMapping": {
+          "destinationName": "AUTHORS",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "overflowBehaviour": "ABORT"
+        }
+      },
+      "Characters": {
+        "toJsonMapping": {
+          "destinationName": "CHARACTERS",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "overflowBehaviour": "ABORT"
+        }
+      },
+      "Cover": {
+        "toVarcharMapping": {
+          "destinationName": "COVER",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      },
+      "Genre": {
+        "toVarcharMapping": {
+          "destinationName": "GENRE",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      },
+      "Pages": {
+        "toDecimalMapping": {
+          "destinationName": "PAGES",
+          "key": "none",
+          "required": false,
+          "overflowBehaviour": "ABORT",
+          "notNumericBehaviour": "ABORT",
+          "decimalPrecision": 20,
+          "decimalScale": 0
+        }
+      },
+      "Price": {
+        "toVarcharMapping": {
+          "destinationName": "PRICE",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      },
+      "Title": {
+        "toVarcharMapping": {
+          "destinationName": "TITLE",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      }
+    }
+  },
+  "$schema": "https://schemas.exasol.com/edml-1.3.0.json"
+}
+```
+
+The generator writes the mapping without formatting. For easier viewing and editing use an editor / IDE with good JSON support and format the file. Good editors also help with editing because they can give you code-completion based on the JSON-schema.
```

Comment:
```suggestion
The generator writes the mapping without formatting. For easier viewing and editing use an editor / IDE with good JSON support and format the file. Good editors also help with editing because they can give you code completion based on the JSON schema.
```